src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -61,7 +61,17 @@ protected ActionResult execute() {
                 log.severe(TeammatesException.toStringWithStackTrace(e));
             }
 
-            statusToUser.add(new StatusMessage(Const.StatusMessages.FEEDBACK_SESSION_ADDED, StatusMessageColor.SUCCESS));
+            String statusMessage;
+
+            if (Const.ParamsNames.FEEDBACK_SESSION_STANDARD.equals(feedbackSessionType)) {
```

Comment:
Emmm i m thinking (coz I interpreted the meaning of `feedbackSessionType` class wrongly...I thought it refers to the type of templates used but actually it refers to if the feedback session is for the creator only :cactus) so do you think we should avoid the use of the name `feedbackSessionType` to refer to the type of template used?

Reply:
Yes that's a very curious thing I noticed too - the meaning of a `FeedbackSessionType` is not very well defined. If you look at the enum alone, it seems like there was an intention to see the different templates as types of `FeedbackSessionType` as well, which can be correct as well if you see a feedback template as inheriting from a standard feedback session type (i.e. a template is a standard feedback session type is a feedback session type). 

But to simplify matters, I do agree with you - let's just treat the templates as separate from `FeedbackSessionType`s. Anyway Standard is not a template so there's no conflict there :P

Reply:
@sukanta-27 Thanks for linking that discussion here! It's very much relevant to the topic at hand so @wenmogu shall we just leave it as it is for now?

Reply:
Thx @sukanta-27 for the info! I see theres no such think as `FeedbackSessionType` anymore. However just to make it clearer I feel that we shld still be more specific abt the name. What abt we follow @sukanta-27 's naming convention and name them as something in line with `sessionTemplateType` ?

Reply:
@wenmogu I was suggesting that if this PR doesn't explicitly need renaming (e.g if the use of feedbackSessionType is causing such confusions that it is harder to review or code) then to avoid renaming, as I have already done it in #8518 (It was kind of easier for me as in this pr, as we need `feedbackSessionType` anymore, so after I removed the attribute, the renaming part was easy, but in this PR @joanneong will have to study which `feedbackSessionType` is used for which purpose and rename the occurrences that imply template type and not session type, and leave the other parts intact, and all of these will be kind of reinventing the wheel) and not to mention the merge conflicts that one of us will have to solve depending whose PR is merged first :laughing:

Reply:
hi @sukanta-27 I think I understand what you mean. Thx for the input! However to me names like `FEEDBACK_SESSION_TEAMEVALUATION` are unclear in their meaning and have to be changed. I know that after the elimination of `FeedbackSessionType`, altho theres nothing to "compete" with these names for meaning, however, at the first sight, ppl will not know these variables are referring to the template types. Just to make the future contributors' life easier when they see it, I think we can make the meaning more obvious by showing the word `template` inside.

As for your concern of merge conflict, I guess that's inevitable as @joanneong has extracted the string "STANDARD" and "TEAMEVALUATION" , so even if she leaves these names like what they r now, there is still gonna be conflict. 

What is ur opinion here @joanneong

Reply:
>However to me names like `FEEDBACK_SESSION_TEAMEVALUATION` are unclear in their meaning and have to be changed.

@wenmogu Sorry for not being very clear, actually I have no issues in renaming these names (e.g changing `FEEDBACK_SESSION_TEAMEVALUATION` to `FEEDBACK_SESSION_TEMPLATE_TEAMEVALUATION`), what I was suggesting was to leave the variables like `feedbackSessionType` as it is, so that if this PR getting merged first, then I will modify the changes made here to `sessionTemplateType` in my PR (as it will definitely throw error when seeing `feedbackSessionType`) and if #8518 gets merged first, then @joanneong will modify **only** her changes accordingly. This way, we don't have to be concerned about all the occurrences of `feedbackSessionType`, but only the changes made in this PR.

>As for your concern of merge conflict, I guess that's inevitable as @joanneong has extracted the string "STANDARD" and "TEAMEVALUATION" , so even if she leaves these names like what they r now, there is still gonna be conflict.

Though it was not much of a concern (As there will be conflicts while working on big PRs), but trying to find a way to minimize the work for one of us I guess :P Anyway you do have a point so I will leave you both to decide the best course of action.

Reply:
ohhh i see. Sorry @sukanta-27 I misinterpreted what you meant. In that case I still suggest to change the variable name from `feedbackSessionType` (only the variables with that names which actually mean the template types) to remind herself and other reviewers of this pr (or even other contributors in case this pr is merged first). 

Thank u very much for the inputs @sukanta-27 🙏 all the best to u and ur PR 🙏

Reply:
Personally I would prefer not changing at the moment (imagine if after @sukanta-27 's #8518 gets reviewed, and the name gets changed...). But I will still do it for now :P